surfer/tests/test_viz.py

```diff
@@ -1,18 +1,18 @@
-import numpy as np
 import os
 import os.path as op
 from os.path import join as pjoin
-import re
 import shutil
-import subprocess
-from nose.tools import assert_equal
-from numpy.testing import assert_raises, assert_array_equal
 from tempfile import mkdtemp, mktemp
+
+import imageio
```

Comment:
I added it as a dependency to `setup.py` to make usage seamless (it's on PYPI). If we remove that I assume we would also need some instructions about installing the dependency...